configuration/pih/scripts/visit/encounterTypeConfig.js

```diff
@@ -1165,7 +1165,7 @@ angular.module("encounterTypeConfig", [])
             var encounterType = encounterTypes[uuid];
 
             if (encounterType == null) {
-              return null;
```

Comment:
Oh, cool, got, thanks!  I guess we must never have relied on the default before now?

Reply:
Seems like it, yeah